libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -2189,6 +2189,63 @@ MQTTStatus_t MQTT_DeserializeAck( const MQTTPacketInfo_t * pIncomingPacket,
     return status;
 }
 
+/*-----------------------------------------------------------*/
+
+MQTTStatus_t MQTT_GetSubAckPayload( MQTTPacketInfo_t * pSubackPacket,
```

Comment:
The first parameter can be `const`
```suggestion
MQTTStatus_t MQTT_GetSubAckPayload( const MQTTPacketInfo_t * pSubackPacket,
```